### ISSUE TITLE
Creating Great Slides

### ISSUE THREAD

**Issue by emmairwin** [emmairwin/leadership_toolkit#15]
Standards and templates for developing content [can be found here](https://wiki.mozilla.org/Participation/Projects/Leadership_toolkit#Work_To_Date).

**Comment by emmairwin** [comment]
Template, based on work by @jgmac1106 
https://docs.google.com/document/d/1dqmAyKaa7m4dtSlKW2C3Eg1hCcLQfiFGhHdDDM9AVPU/edit#

Ready for review ?

**Comment by emmairwin** [comment]
tagging for our call tomorrow, but feel free to add descriptions etc  before

**Comment by emmairwin** [comment]
Can't wait to see this one

**Comment by emmairwin** [comment]
@jgmac1106 I published this

https://leadership.mozilla.community/method/bring-the-bling-creating-awesome-slides/

We haven't been using the QA checklist, and I wonder how you would feel about trying that with this content?  I started, but then realized since you wrote it, probably better reviewed by you?  The one thing I realize we don't have in this one (and many others) is 'there are many ways for a learner to evaluate their progress'.

https://docs.google.com/spreadsheets/d/1VqJXy3bslt1yQCQ7Q1NiMXcWUQbaE2AbLtHXqTh_X2A/edit#gid=0
